scripts/utils/logSummary.js

```diff
@@ -0,0 +1,145 @@
+const getCliArgs = require('./getCliArgs');
+const boxify = require('./boxify');
+const colorize = require('./colorize');
+
+const stepMap = {
+    'checkout-repo': {
+        order: 1,
+        label: 'Checkout Repository',
+        include: true,
+    },
+    'setup-node': {
+        order: 2,
+        label: 'Setup Node',
+        include: true,
+    },
+    'variables': {
+        order: 3,
+        label: 'Extract Variables',
+        include: true,
+    },
+    'configure-aws': {
+        order: 4,
+        label: 'Configure AWS credentials',
+        include: true,
+    },
+    'start-metrics': {
+        order: 5,
+        label: 'Updater Start Metrics',
+        include: false,
+    },
+    'cache': {
+        order: 6,
+        label: 'Cache node_modules',
+        include: true,
+    },
+    'authenticate': {
+        order: 7,
+        label: 'Authenticate siteId and secretKey',
+        include: true,
+    },
+    'install': {
+        order: 8,
+        label: 'Install Packages',
+        include: true,
+    },
+    'build': {
+        order: 9,
+        label: 'Build Bundle',
+        include: true,
+    },
+    'test': {
+        order: 10,
+        label: 'Run Tests',
+        include: true,
+    },
+    'snapfu-recs-sync': {
+        order: 11,
+        label: 'Snapfu Recs Sync',
+        include: true,
+    },
+    'lighthouse': {
+        order: 12,
+        label: 'Lighthouse Tests',
+        include: true,
+    },
+    'tag': {
+        order: 13,
+        label: 'Git Tag',
+        include: false,
+    },
+    'pr-comment': {
+        order: 14,
+        label: 'PR Comment',
+        include: true,
+    },
+    'lighthouse-metrics': {
+        order: 15,
+        label: 'Lighthouse Metrics',
+        include: true,
+    },
+    's3-upload': {
+        order: 16,
+        label: 'Copy Files to S3',
+        include: true,
+    },
+    'invalidation': {
+        order: 17,
+        label: 'Invalidate Files',
+        include: true,
+    },
+    'metrics': {
+        order: 18,
+        label: 'Conclusion Metrics',
+        include: true,
+    },
+}
+
+const conclusions = {
+    success: '✅',
+    failure: '❌',
+    skipped: '⛔',
```

Comment:
Looking at the output now, I think we should change the "skipped" emoji to something a bit more subtle so that it is not confused with a failure. Maybe a dash (➖) or something?

Reply:
I agree